static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -201,11 +197,18 @@ function useReplayData({eventSlug, orgId}: Options): Result {
     setState(INITIAL_STATE);
 
     try {
-      const [event, attachments] = await Promise.all([fetchEvent(), fetchRRWebEvents()]);
+      const [replayResponse, attachments] = await Promise.all([
+        fetchReplay(),
+        fetchRRWebEvents(),
+      ]);
 
       setState(prev => ({
         ...prev,
-        event,
+        replayRecord: {
+          ...replayResponse.data,
+          started_at: new Date(replayResponse.data.started_at),
+          finished_at: new Date(replayResponse.data.finished_at),
```

Comment:
We should follow-up and make sure these are camelCase (from backend)

Reply:
Yes

We can follow the changes here: https://github.com/getsentry/sentry/pull/36943#discussion_r937972981
And in the blueprint too i expect: https://github.com/getsentry/replay-backend/pull/15/files